bazel/local_jdk_libjvm.bzl

```diff
@@ -0,0 +1,60 @@
+# Copyright 2021 Code Intelligence GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def _find_recursively_under_path(repository_ctx, path, basename):
+    result = repository_ctx.execute([
+        repository_ctx.which("sh"),
+        "-c",
+        """find -L "{path}" -name "{basename}" | head -1""".format(
+            path = path,
+            basename = basename,
+        ),
+    ])
+    if result.return_code != 0:
+        return None
+    file_path = result.stdout.strip()
+    if not file_path:
+        return None
+    return repository_ctx.path(file_path)
+
+LIBJVM_NAMES = [
+    "libjvm.dll",
```

Comment:
Could this rule work on Windows? The find command above and the unix path separators below would probably need some adjustments for Windows, no? Maybe we can drop `libjvm.dll` since we don't support Windows?

Reply:
You are right, having it here doesn't make much sense without a `cmd` command to find it.